src/atoms/tags/tag.stories.tsx

```diff
@@ -1,21 +1,53 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { ComponentMeta, ComponentStory } from '@storybook/react';
 import { Tag } from './tag';
 
 export default {
   title: 'Atoms/Tags',
   component: Tag,
   argTypes: {
+    type: {
+      control: {
+        type: null,
+      },
+    },
   },
 } as ComponentMeta<typeof Tag>;
 
-const Template: ComponentStory<typeof Tag> = (args) => <Tag {...args} />;
-export const Primary = Template.bind({});
-
-Primary.parameters = {
-  backgrounds: { default: 'dark' },
+const Template: ComponentStory<typeof Tag> = (args) => {
+  const [isActive, setIsActive] = useState<boolean>(false);
+  return (
+    <Tag {...args} isOn={isActive} onClick={() => setIsActive(!isActive)} />
+  );
 };
-Primary.args = {
-  name: 'Tag',
+export const Tag1 = Template.bind({});
+export const Tag2 = Template.bind({});
+export const Tag3 = Template.bind({});
+export const Tag4 = Template.bind({});
+export const Tag5 = Template.bind({});
+export const Tag6 = Template.bind({});
+
+Tag1.args = {
+  name: 'Tag 01',
```

Comment:
@ArinCantCode so are we wanting to use those categories nft, defi, privacy etc? I guess these might change, be added t at some point though?

Reply:
yeach i think we can use them, its just to show different variations of tag and how they look. For example i noticed that the current tags on newsdesk can just have two letters `lp` if its just two letters the tag is pretty much circle. Not that its something bad but its nice to see different variations like that because Michal might want to adjust something after reviewing